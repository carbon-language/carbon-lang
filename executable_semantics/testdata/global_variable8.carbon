// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
// Exceptions. See /LICENSE for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

// Test that a global variable may not depend on a later global.
// Error expected.

var Int x = y;

var Int y = 0;

fn main() -> Int {
  return x;
}
