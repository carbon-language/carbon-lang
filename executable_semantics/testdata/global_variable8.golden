RUNTIME ERROR: 8: could not find `y`
EXIT CODE: 255
